src/v0/destinations/klaviyo/transform.js

```diff
@@ -274,9 +293,100 @@ const process = async (event) => {
   return result;
 };
 
+const batchEvents = (successRespList) => {
+  const batchedResponseList = [];
+  const identifyResponseList = [];
+  successRespList.forEach((event) => {
+    if (event.message.length === 2) {
+      identifyResponseList.push(event.message[0]);
+      event.message = event.message[1];
+    } else {
+      event.message = event.message[0];
```

Comment:
You are not sending an array when there is only one value. For reference see line #126.

Reply:
Yes, that is done intentionally. Those events will not come to this function